### ISSUE TITLE
Multiplayer Glicko2Implementation

### ISSUE THREAD

**Issue by CesMak** [sublee/glicko2#10]
Hey there,
I would like to use glicko2 for a multiplayer card game like witches or wizzard from amigo. Lets say you have 4 players that play against each other and lets say at the end of the game player 1 wins and all other player loose - in glicko2 as far as I understood your implementation I only can compute a rating between two players is that right?

`        rate(rating_player1, [(WIN, rating_player2)])`

Is there a fast hack to go for multiplayer? What would you suggest?
Taking the mean of all other players? as rating_player2?

**Comment by gbtami** [comment]
Maybe you should try https://trueskill.org/ instead.

**Comment by CesMak** [comment]
Yeah I saw that but you cannot use trueskill commercially and it has some well known problems.``

Anyways I wonder why this is not the same:

```python
    env = Glicko2(tau=0.5)
    r1 = env.create_rating(1500, 200, 0.06)
    r2 = env.create_rating(1500, 200, 0.06)
    r3 = env.create_rating(1500, 200, 0.06)

    print("Initial 2 new players:", printLevels([r1, r2]), r1)
    r11 = env.rate(r1, [(WIN, r2)])
    r111 = env.rate(r11, [(LOSS, r3)])
    print(printLevels([r111]), r111)

    r_direct = env.rate(r1, [(WIN, r2), (LOSS, r3)])
    print(printLevels([r_direct]), r_direct)
```

```
Initial 2 new players: [8300.0, 8300.0] (mu=1500.000, phi=200.000, sigma=0.060)
[8810.8053] (mu=1499.474, phi=165.683, sigma=0.060)
[8825.4008] (mu=1500.000, phi=164.973, sigma=0.060)
```

**Comment by CesMak** [comment]
This is Not usefull for commercial Projects I think I will test how it works using the mean.... Am 07.08.2020 12:01 schrieb Bajusz Tamás <notifications@github.com>:
Maybe you should try https://trueskill.org/ instead.
